### PR TITLE
DOC: Remove extra indents in documents

### PR DESCRIPTION
**Pull request by ghost**
Fixed the incorrect blockquote markups in some documents.

[skip cirrus] [skip travis] [skip azp] [skip actions]

### CODE REVIEW

File: doc/source/dev/internals.rst [hunk 1]
```diff
@@ -19,21 +19,21 @@ items. NumPy also contains a significant set of data that describes how to
...
- 5) The byte order of the data (which may not be the native byte order).
- 6) Whether the buffer is read-only.
- 7) Information (via the :class:`dtype` object) about the interpretation of the
-    basic data element. The basic data element may be as simple as an int or a
-    float, or it may be a compound object (e.g.,
-    :term:`struct-like <structured data type>`), a fixed character field,
-    or Python object pointers.
- 8) Whether the array is to be interpreted as :term:`C-order <C order>`
-    or :term:`Fortran-order <Fortran order>`.
+1) The basic data element's size in bytes.
```
Comment (charris) [inline doc/source/dev/internals.rst:22]: Should the ")" be replaced by "."?

Comment (charris) [reply doc/source/dev/internals.rst:22]: It seems to work, but is uncommon.

**Review by charris — APPROVED** [review]

File: doc/source/dev/internals.rst (hunk 1 above)
Comment (F3eQnxN3RriK) [reply doc/source/dev/internals.rst]: Thanks, fixed it.

**Comment by charris** [comment]
Thanks @F3eQnxN3RriK .

**Pull request merged**